SPECS/krb5/krb5.spec

```diff
@@ -46,30 +49,30 @@ autoconf &&
     --prefix=%{_prefix} \
     --bindir=%{_bindir} \
     --libdir=%{_libdir} \
-    --sysconfdir=/etc \
-        --localstatedir=/var/lib \
-        --with-system-et         \
-        --with-system-ss         \
-        --with-system-verto=no   \
-        --enable-dns-for-realm   \
-        --enable-pkinit          \
-        --enable-shared          \
-        --without-tcl
+    --sysconfdir=%{_sysconfdir} \
+    --localstatedir=%{_sharedstatedir} \
+    --with-system-et         \
+    --with-system-ss         \
+    --with-system-verto=no   \
+    --enable-dns-for-realm   \
+    --enable-pkinit          \
+    --enable-shared          \
+    --without-tcl
 make %{?_smp_mflags}
 
 %install
 cd src
 [ %{buildroot} != "/"] && rm -rf %{buildroot}/*
 make install DESTDIR=%{buildroot}
-find %{buildroot}/%{_libdir} -name '*.la' -delete
+find %{buildroot} -type f -name "*.la" -delete -print
```

Comment:
This find command is a little bit different than the one it is replacing. Adding a note to confirm if this still does the intended behavior.

Reply:
It is :). In this case the `.la` files were only under the `libdir` for the spec, but the linter replaced it with a more generic version that will still achieve the same desired results.